src/Serilog/Formatting/Display/OutputProperties.cs

```diff
@@ -72,7 +72,7 @@ public static class OutputProperties
             result[TimestampPropertyName] = new ScalarValue(logEvent.Timestamp);
             result[LevelPropertyName] = new LogEventLevelValue(logEvent.Level);
             result[NewLinePropertyName] = new LiteralStringValue(Environment.NewLine);
-            result[PropertiesPropertyName] = new LogEventPropertiesValue(logEvent.MessageTemplate, logEvent.Properties);
+            result[PropertiesPropertyName] = new LogEventPropertiesValue(logEvent.MessageTemplate, logEvent.Properties, result);
```

Comment:
I would just add a comment before this line to remind future explorers to make sure this value always comes last so it picks up all the existing properties.